node/src/tasks/bsp_volunteer_mock.rs

```diff
@@ -43,6 +53,24 @@ impl<SHC: StorageHubHandlerConfig> EventHandler<NewStorageRequest> for BspVolunt
             event.fingerprint
         );
 
+        let result = self.inner_handle_event(event).await;
+        if result.is_err() {
+            if let Some(file_key) = &self.file_key_cleanup {
+                let _ = self
```

Comment:
The error can only be `FileNotRegistered` which is normal to expect since we might have 2 parallel requests fail and try to remove the file key.
I will log it for now as an warning.